docs/DeveloperGuide.md

```diff
@@ -171,6 +171,31 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Enhanced Find Logic
+
+The previous implementation find target is limited to name field only. Hence, extending the find function will increase the user experience by enable them easily search `Person` with `phone`
+`email` and `address` field.
+This is done by creating `searchType` model within the `model util` package and extract original `NameCdonstrainsKeywordsPreicate` to form `CdonstrainsKeywordsPreicate` then create different inherit of three different types of predicate.
```

Comment:
```suggestion
This is done by creating `searchType` model within the `model util` package and extracting the original `NameCdonstrainsKeywordsPreicate` to form `CdonstrainsKeywordsPreicate` then create different inherit of three different types of predicate.
```
Consider changing the grammar. The part after `CdonstrainsKeywordsPreicate` sound abit weird

Reply:
thx changed